src/pages/api/curriculum-downloads/index.ts

```diff
@@ -65,13 +65,11 @@ async function getData(opts: {
   const dataWarnings: string[] = [];
 
   try {
-    const curriculumDataUnsorted =
-      await curriculumApi2023.curriculumUnitsIncludeNew({
-        subjectSlug,
-        phaseSlug,
-        examboardSlug: examboardSlug ?? null,
-        state,
-      });
+    const curriculumDataUnsorted = await curriculumApi2023.curriculumUnits({
+      subjectSlug,
+      phaseSlug,
+      ks4OptionSlug: examboardSlug ?? null,
```

Comment:
Should `examboardSlug ?? null` read `ks4OptionSlug ?? null`

Reply:
Yes - will create another PR as it needs to basically be changed everywhere